packages/contracts-bedrock/CONTRIBUTING.md

```diff
@@ -90,16 +91,16 @@ To deploy the smart contracts on a local devnet, run `make devnet-up` in the mon
 
 In order to make sure that we don't accidentally overwrite storage slots, contract storage layouts are checked to make sure spacing is correct.
 
-This uses the `.storage-layout` file to check contract spacing. Run `pnpm validate-spacers` to check the spacing of all contracts.
+This uses the `.storage-layout` file to check contract spacing. Run `just validate-spacers` to check the spacing of all contracts.
```

Comment:
I think this is outdated too and we don't use `.storage-layout` right? I know the text wasn't changed in this PR, but since we're modifying it here anyway it would be good to fix

Reply:
I resolved this